x-pack/plugins/stack_connectors/server/connector_types/jira/service.ts

```diff
@@ -356,72 +347,24 @@ export const createExternalService = (
     }
   };
 
-  const getCapabilities = async () => {
+  const getIssueTypes = async () => {
     try {
       const res = await request({
         axios: axiosInstance,
         method: 'get',
-        url: capabilitiesUrl,
+        url: getIssueTypesUrl,
         logger,
         configurationUtilities,
         connectorUsageCollector,
       });
 
       throwIfResponseIsNotValid({
         res,
-        requiredAttributesToBeInTheResponse: ['capabilities'],
       });
 
-      return { ...res.data };
-    } catch (error) {
-      throw new Error(
-        getErrorMessage(
-          i18n.NAME,
-          `Unable to get capabilities. Error: ${error.message}. Reason: ${createErrorMessage(
-            error.response?.data
-          )}`
-        )
-      );
-    }
-  };
-
-  const getIssueTypes = async () => {
-    const capabilitiesResponse = await getCapabilities();
-    const supportsNewAPI = hasSupportForNewAPI(capabilitiesResponse);
-    try {
-      if (!supportsNewAPI) {
-        const res = await request({
-          axios: axiosInstance,
-          method: 'get',
-          url: getIssueTypesOldAPIURL,
-          logger,
-          configurationUtilities,
-          connectorUsageCollector,
-        });
-
-        throwIfResponseIsNotValid({
-          res,
-        });
-
-        const issueTypes = res.data.projects[0]?.issuetypes ?? [];
-        return normalizeIssueTypes(issueTypes);
-      } else {
-        const res = await request({
-          axios: axiosInstance,
-          method: 'get',
-          url: getIssueTypesUrl,
-          logger,
-          configurationUtilities,
-          connectorUsageCollector,
-        });
-
-        throwIfResponseIsNotValid({
-          res,
-        });
-
-        const issueTypes = res.data.values;
-        return normalizeIssueTypes(issueTypes);
-      }
+      // Cloud returns issueTypes and Data Center returns values
+      const { issueTypes, values } = res.data;
+      return normalizeIssueTypes(issueTypes || values);
```

Comment:
I'm not a big fan of this assumption and I think that is another argument towards having some distinction between Data Center or Cloud.

We expect the API to return one or the other but what if we start getting something like
```
{
    issueTypes: {},
    values: { foo: "bar" }
}
```
?

Then `issueTypes || values` will be `{}`.